api/service_routes/room-management.yaml

```
@@ -3,18 +3,154 @@ info:
   title: API Title
   version: '1.0'
 servers:
-  - url: https://room-management.reserva.emilharvey.ca
+  - url: https://api.reserva.emilharvey.ca
 paths:
-  /test:
+  /rooms/{id}:
     get:
+      tags: 
+        - Room Management
+      summary: Gets a Room object by id
+      description: >
+        ## More Information
+
+        Request for `GET /rooms/{id}` requires an id which will be in the search parameter.
+        Response will be in JSON format with S
+      operationId: getRoomId
+      parameters:
+        - name: id
+          in: path
+          description: User ID
+          required: true
+          schema:
+            type: integer
+            format: int64
       responses:
         '200':
           description: OK
+          content:
+            application/json:
+              schema:
+                properties:
+                  "Room":
+                    $ref: "#/components/schemas/Room"
+    put:
+      tags: 
+        - Room Management
+      summary: Updates a Room object
+      description: >
+        ## More Information
+
+        Request for `PUT /rooms/{id}` requires a body that contains all information of the Room except for its id. 
+        (The id of the room will be auto-incremented).
+        Response body shows the body that contains 
```

The `id` won't be auto incremented. `put` here should be for updating an existing room.


```suggestion
        Updates an existing room (identified by `id`).
```